### PR TITLE
kaf: Add version 0.1.38

### PR DESCRIPTION
**Pull request by nightscape**
Modern CLI for Apache Kafka, written in Go.

### CODE REVIEW

File: bucket/kaf.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        [
+            "kaf.exe",
+            "kaf"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/birdayz/kaf"
+    },
+    "autoupdate": {
+        "url": "https://github.com/birdayz/kaf/releases/download/v$version/kaf_$version_Windows_x86_64.tar.gz",
```
Comment (Ash258) [inline bucket/kaf.json:18]:
```suggestion
        "architecture": {
            "64bit": {
                "url": "https://github.com/birdayz/kaf/releases/download/v$version/kaf_$version_Windows_x86_64.tar.gz"
            }
        },
```

File: bucket/kaf.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    "hash": "ce3cc29362b68f2b8a54c7653c34e1381a760d6135ad014592106b727cf1f391",
+    "bin": [
+        [
+            "kaf.exe",
+            "kaf"
+        ]
+    ],
+    "checkver": {
+        "github": "https://github.com/birdayz/kaf"
+    },
```
Comment (Ash258) [inline bucket/kaf.json:16]:
```suggestion
    "version": "0.1.38",
    "description": "Modern CLI for Apache Kafka",
    "homepage": "https://github.com/birdayz/kaf",
    "license": "Apache-2.0",
    "architecture": {
        "64bit": {
            "url": "https://github.com/birdayz/kaf/releases/download/v0.1.38/kaf_0.1.38_Windows_x86_64.tar.gz",
            "hash": "ce3cc29362b68f2b8a54c7653c34e1381a760d6135ad014592106b727cf1f391"
        }
    },
    "bin": "kaf.exe",
    "checkver": "github",
```

**Comment by nightscape** [comment]
@Ash258 thanks for you suggestions, applied!
I always keep on forgetting to scope under 64bit architecture...

**Pull request merged**